common.lic

```diff
@@ -105,7 +103,13 @@ module DRC
 
     case result
     when 'You feel about'
-      release_invisibility
+      # Reduntant copy of DRCA.release_invisibility to avoid deadlocking from including commona-arcana
```

Comment:
Don't mean redundant in a negative sense btw, just saying DRY is too rigid in this case :]